### ISSUE TITLE
Howto expose nested class in JS API?

### ISSUE THREAD

**Issue by Borecjeborec1** [r0x0r/pywebview#1110]
### Specification
- pywebview version: 4.0.2
- operating system: win10
- web renderer: edgechromium

### Description  

Hi!

Is there a way of exposing more python APIs to js? Or creating some nested ones? 

In the frontend JS I would like to be able to use `pywebview.api.Fs.readFile()` or  `pywebview.api.Window.getSize()` 
I know I can use some object mapping in the JS code so I can map the function I want and nest it inside other object. But for this would require additional JS code with the need of knowing exact names of the exposed functions. So this approach is not optimal for me. 
For that reason I am looking for something like this:
```python
webview.create_window("test","index.html",js_api=(FsApi(),WindowApi()))
```
or maybe this:
```python
class Api:
   class Fs:
      def readFile(_path):
           # readfile implementation 
   class Window:
      def getSize():
           # getWindowSize implementation 
webview.create_window("test","index.html",js_api=(Api()))
```
But none of those work. Maybe I am missing some easier variation of doing so...

I couldn't really find a solution for this. I was also trying with expose_function but this only expose function (as the name suggest :D) and can't expose the whole class.

_TL;DR: I am looking for a way of exposing whole class instead of function or nested class with the js_api parameter so I can use `pywebview.api.someClass.someMethod()`._

### Practicalities

- YES I am willing to work on this issue myself.

**Comment by r0x0r** [comment]
Subclasses are not taken into account when parsing, but this could be done. Feel free to work on this, if you like.

**Comment by r0x0r** [comment]
A better implementation would be to add nested class support to util -> parse_api_js.
